docs/cosmology/FLRWCheatSheet.rst

```diff
@@ -0,0 +1,56 @@
+:orphan:
+
+==================================
+FLRW Cosmology Method Cheat Sheet
+==================================
+ A class describing an isotropic and homogeneous (Friedmann-Lemaitre-Robertson-Walker) cosmology.
+
+ Refer to `FLRW documentation for detailed information <http://docs.astropy.org/en/latest/api/astropy.cosmology.FLRW.html/>`_.
+
```

Comment:
These should links to the classes or the pages and not links to the online documentation.  Please see some of the other documentation pages for examples for how this should be linked.

Reply:
For example

    `~astropy.cosmology.FLRW`

rather than the link

Reply:
Thanks, I have changed the link to the appropriate format